firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -1050,6 +1050,93 @@ extension String {
     }
 }
 
+// MARK: - Micro survey
+extension String {
+    public struct Microsurvey {
+        public struct Prompt {
+            public static let TitleLabel = MZLocalizedString(
+                key: "Microsurvey.Prompt.TitleLabel.v127",
```

Comment:
FYI - these versions are tagged `v127` because these are when the strings are going in, but the feature itself is targeted for `v129`